ros/src/models/chrono/ros_chrono/launch/steering.launch

```diff
@@ -10,5 +10,8 @@
   <rosparam command="load" file="$(arg chrono_params_path)"/>
  
   <node name="Reference" pkg="traj_gen_chrono" type="traj_gen_chrono"/>
+  <test test-name="Reference_test" pkg="traj_gen_chrono" type="traj_gen_chrono"/>
+
   <node name="Chronode" pkg="ros_chrono" type="steering_controller" cwd="node" />
+  <test test-name="Chronode_test" pkg="ros_chrono" type="steering_controller" />
```

Comment:
@Scarabrine It does not make sense to me that we would be running a test inside this launch file. Testing should be done separately from running software

Reply:
Yes, I also doubt if I should put the test node in the real launch file. Although it will only be executed when it's called by 'catkin_make run_tests', making test code in real demo is not a proper way. I will further study how to synthesize them together

Reply:
ahh! cool, I did not know that! If that is that case great!